tasks_examples/OALL_tasks.txt

```diff
@@ -0,0 +1,117 @@
+lighteval|arabic_mmlu:abstract_algebra|5|1
```

Comment:
You'll need to edit to use `community` as a suite

Reply:
done